src/clarity-angular/wizard/wizard.html

```diff
@@ -12,7 +12,7 @@
     [clrModalSkipAnimation]="stopModalAnimations"
     [clrModalGhostPageState]="ghostPageState"
     [clrModalOverrideScrollService]="isStatic"
-    clrModalPreventClose="true"
+    [clrModalPreventClose]="true"
```

Comment:
so isn't of passing a `string` we are now passing a `boolean`. Has the type changed or was this a bug before?